library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -309,7 +309,23 @@ public void setLoopCount(int loopCount) {
         }
 
         if (loopCount == LOOP_INTRINSIC) {
-            maxLoopCount = decoder.getLoopCount();
+            int netscapeIterationCount = decoder.getNetscapeIterationCount();
```

Comment:
This method should have been part of the decoder, people sometimes use the decoder directly, so any time they need to get the real loop count, they would have to copy this code from GifDrawable into wherever they need it.